cpu-profiling/cl-position-handler/README.md

```diff
@@ -0,0 +1,357 @@
+# Profile the Central-Ledger Position Handler in isolation
+
+## Status
+
+| Central Ledger Version |  Date  | Status  | Next  | Notes  |
+|---|---|---|---|---|
+| 17.0.3 | 2023-08-25 | **The `Position Handler` has achieved a maximum observed maximum throughput of around `163 Op/s` @ `6.37 ms` average duration for `action=prepare` messages and `394 Op/s` @ `3.81 ms` average duration for `action=commit` messages as baseline with the system configuration specified in 'Testing Environment' section.<br><br>**An overview of the observations are summarized as follows:** <br> 1. No considerable issues detected in profiling with Chrome debugger <br>2. No issues detected  with Clinic.js bubbleprof analyser <br>3. If CACHE is disabled, there are `13` queries are being executed during processing a `action=prepare` message. And `8` queries during `action=commit` message including BEGIN and COMMITs.<br>4. For `5000` messages the total queries for `action=prepare` are `65000` if CACHE is disabled. This matches with the query count in single transfer (i.e. 5000x13)<br>5. If CACHE is enabled, for `5000` messages the total queries for `action=prepare` are reduced to `45000` 🔽<br>6. The indexes are properly set and no slow queries detected<br><br> **Observed areas that are limiting throughput and scalability:**<br> 1. `Batch processing` implementation is broken and can not be enabled which could improve performance <br> 2. With current setup (kafka key and partition combination), `Scalability` of position handler is not possible for a single DFSP<br><br> _Note: We are not able to determine any single cause of the bottleneck at this time, the load should be handled through batching and scalability._ | See [#follow-up-stores](#follow-up-stories) |   |
```

Comment:
1. Include the number of queries that occur when caching is enabled.
2. Include findings for "scalability" when more than now DFSP is being processed.
3. Include high-level findings of PRISM

Reply:
Done